### PR TITLE
Add entry

### PR DESCRIPTION
**Pull request by JackPadalino**
Now updating the front end with newly added entries

### CODE REVIEW

File: src/Components/Entry/NewEntry.tsx [hunk 1]
```diff
@@ -16,8 +19,9 @@ interface props {
 };
 
 const NewEntry = ({accountId}:props) => {
+    const dispatch = useDispatch();
     const [entryType, setEntryType] = useState<string>("User");
-    const [amount, setAmount] = useState<number|null>(null);
+    const [amount, setAmount] = useState<number|string>("");
```
Comment (AOKaizo) [inline src/Components/Entry/NewEntry.tsx:24]: string?

Comment (AOKaizo) [reply src/Components/Entry/NewEntry.tsx:24]: Null?

Comment (JackPadalino) [reply src/Components/Entry/NewEntry.tsx]: Good point! We wouldn't want anyone adding a string to this form. I'll set to null.

**Review by AOKaizo — APPROVED** [review]
Great Changes!

**Pull request merged**